tests/test_type_alias_type.py

```diff
@@ -400,8 +400,8 @@ class MyModel(BaseModel):
     assert MyModel(my_int_seq=range(1, 4)).my_int_seq == [1, 2, 3]
 
     assert MyModel.model_json_schema() == {
-        '$defs': {'MySeq_int_': {'items': {'type': 'integer'}, 'type': 'array'}},
-        'properties': {'my_int_seq': {'$ref': '#/$defs/MySeq_int_'}},
+        '$defs': {'MyIntSeq': {'items': {'type': 'integer'}, 'type': 'array'}},
+        'properties': {'my_int_seq': {'$ref': '#/$defs/MyIntSeq'}},
```

Comment:
No I dont think so, it should just make the json schema names more clear for TypeAliasType so it uses the user defined names. Comes from the "proper" usage of `GenerateSchema` `_Definitions`

Reply:
Right, but I'm concerned that folks were dependent upon the old naming pattern for parametrized generic models in JSON schema

Reply:
This PR is already so large that I'd prefer we remove this specific change. Feel free to open an issue requesting that this be changed, and we can discuss further there 👍

Reply:
I dont think its easy to just remove as the change came from the fundamental issues with the generics handling. So it would require some further hack just for the sake of getting name like it previously was.

Reply:
Also looking this came from very recent PR changing type alias handling. So I dont think anyone relies on this